requirements.txt

```diff
@@ -4,5 +4,5 @@ pytest
 pandas
 scikit-image
 scipy
-opencv-python
-pyyaml==5.4.1
\ No newline at end of file
+opencv-contrib-python
```

Comment:
Hi @JasonGellis, these changes in line 7 and 8 fixed the broken test. Will explain later today.